Sources/Networking/Responses/CustomerCenterConfigResponse.swift

```diff
@@ -132,6 +132,7 @@ struct CustomerCenterConfigResponse {
     struct Support {
 
         let email: String
+        let shouldWarnCustomerToUpdate: Bool?
```

Comment:
Hmm I'm thinking... the original idea was that, if the backend sent the `lastPublishedAppVersion`, the SDK would consider it "enabled" implicitly. The backend shouldn't send that version if it's disabled. So this property would be unnecessary I think?